assets/js/atomic/blocks/product/image/attributes.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+export const blockAttributes = {
```

Comment:
Defining these separately so the frontend.js can validate attributes against them.